### PR TITLE
Add description of collections and become_exe keywords

### LINKED ISSUE #68024: "collections" keyword is UNDOCUMENTED
##### SUMMARY

There is no documentation of "collections" keyword in Keywords page: https://docs.ansible.com/ansible/2.9/reference_appendices/playbooks_keywords.html

##### ISSUE TYPE
- Documentation Report

##### ANSIBLE VERSION

```paste below
2.9.5
```

### PR DESCRIPTION
**Pull request by samdoran**
##### SUMMARY

The descriptions were just missing.

Fixes #68024

##### ISSUE TYPE

- Docs Pull Request
##### COMPONENT NAME

`docs/docsite/keyword_desc.yml`

### CODE REVIEW

File: docs/docsite/keyword_desc.yml [hunk 1]
```diff
@@ -7,6 +7,7 @@ always: List of tasks, in a block, that execute no matter if there is an error i
 any_errors_fatal: Force any un-handled task errors on any host to propagate to all hosts and end the play.
 async: Run a task asynchronously if the C(action) supports this; value is maximum runtime in seconds.
 become: Boolean that controls if privilege escalation is used or not on :term:`Task` execution.
+become_exe: Path to the executable used to elevate privileges.
```
Comment (jborean93) [inline docs/docsite/keyword_desc.yml]: Might be a good idea to link to the become plugin page for this or above saying this is implemented by the become plugin itself. I know `runas` has no use for this value.

**Review by bkmeneguello — APPROVED** [review]

File: docs/docsite/keyword_desc.yml [hunk 2]
```diff
@@ -17,6 +18,11 @@ check_mode: |
 
     .. seealso:: :ref:`check_mode_dry`
 
+collections: |
+    List of collection namespaces to search for modules.
+
+    .. sealso:: :ref:`using_collections_in_playbook:`
```
Comment (bcoca) [inline docs/docsite/keyword_desc.yml]:
```suggestion
    .. seelso:: :ref:`using_collections_in_playbook:`
```

File: docs/docsite/keyword_desc.yml [hunk 3]
```diff
@@ -17,6 +18,11 @@ check_mode: |
 
     .. seealso:: :ref:`check_mode_dry`
 
+collections: |
+    List of collection namespaces to search for modules.
+
+    .. seelso:: :ref:`using_collections_in_playbook:`
```
Comment (s-hertel) [inline docs/docsite/keyword_desc.yml]:
```suggestion
    .. seealso:: :ref:`using_collections_in_playbook:`
```

File: docs/docsite/keyword_desc.yml [hunk 4]
```diff
@@ -17,6 +18,11 @@ check_mode: |
 
     .. seealso:: :ref:`check_mode_dry`
 
+collections: |
+    List of collection namespaces to search for modules.
```
Comment (s-hertel) [inline docs/docsite/keyword_desc.yml]:
```suggestion
    List of collection namespaces to search for modules and roles.
```
Maybe this should also note that tasks in a role do not inherit the playbook keyword (or that the collections keyword needs to be used in meta/main.yml to apply to tasks within a role?).

Comment (samdoran) [reply docs/docsite/keyword_desc.yml]: Good point. I will add a note explaining this.

**Comment by acozine** [comment]
Thanks @samdoran, this looks good! CI ran today and passed, so it's cleared to merge.

**Comment by acozine** [comment]
Dammit! Clicked the wrong button.

**Comment by tpo** [comment]
Thanks all, this also fixes the [collections link in devel](https://docs.ansible.com/ansible/devel/galaxy/user_guide.html) noted in https://github.com/ansible/ansible/issues/68024#issuecomment-599531037

**Pull request merged**